settings/provisioner/openstack/openstack.spec

```diff
@@ -0,0 +1,60 @@
+---
+subparsers:
+    openstack:
+        help: Provision systems using Ansible OpenStack modules
+        groups:
+            - title: cloud
+              options:
+                  cloud:
+                      type: Value
+                      help: The cloud which the OpenStack modules will operate against. Cloud setup instructions: http://docs.openstack.org/developer/os-client-config/#config-files
+                      required: yes
+            - title: prefix
+              options:
+                  prefix:
+                      type: Value
+                      help: An prefix which would be contacted to each provisioned resource
```

Comment:
help: "If not provided, all resource names will be prefixed with a common random string"